ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Block.hs

```diff
@@ -56,6 +56,8 @@ import qualified Shelley.Spec.Ledger.API as SL
 
 import           Ouroboros.Consensus.Shelley.Eras
 
+import           Ouroboros.Consensus.HardFork.Combinator
```

Comment:
This belongs in the same import group above as `Ouroboros.Consensus.Block` (see the `StyleGuide.md` file for more explanation)